roles/deploy/tasks/main.yml

```diff
@@ -138,7 +138,7 @@
   tags: skip_ansible_lint
 
 - name: delete old release files
-  shell: ls -1tr | head -n -10 | xargs -d '\n' rm -rf --
+  shell: set -o pipefail && ls -1tr | head -n -10 | xargs -d '\n' rm -rf --
```

Comment:
Did you test this? I read that the default shell is `sh` which doesn't support the pipefail option. So I executed bash:
https://github.com/openfoodfoundation/ofn-install/blob/459c3d01495a94dc45650be7822b0ff0deb7ddf6/roles/deploy/tasks/main.yml#L122